src/less/components/search/index.less

```diff
@@ -1,4 +1,117 @@
 @import (reference) 'mixins';
+@import (reference) './SearchControl';
+
+// ----- LAYOUT -----
+/*
+    auto    1fr    auto
+  +-------+-------+-------+
+  | lbl   | lbl   | lbl   | auto
+  +-------+-------+-------+
+  | icon  | input | btn   | auto
+  +-------+-------+-------+
+  | extra | extra | extra | auto
+  +-------+-------+-------+
+*/
+
+
+hx-search-control {
+  -ms-grid-column-align: start;
+  -ms-grid-columns: 2rem 1fr auto;
+  -ms-grid-rows: auto auto auto;
+  display: -ms-grid;
+  display: grid;
+  grid-template-areas:
+    'lbl   lbl   lbl'
+    '.     input btn'
+    'extra extra extra';
+  grid-template-columns: 2rem 1fr auto;
+  grid-template-rows: auto auto auto;
+
+  > input[type="search"] {
+    -moz-appearance: textfield;
+    -webkit-appearance: textfield;
+    -ms-grid-column: 2;
```

Comment:
Add `appearance: textfield;` for future compatibility?

Wouldn't want a browser updating to standard syntax to break the appearance.

Reply:
Sounds good.  `appearance: textfield;` added.